apps/stats/src/views/Stats/Overview/Overview.tsx

```diff
@@ -221,7 +221,7 @@ const Overview: React.FC = () => {
                     <HelpCard
                         description='Find out how to review the performance of your content and get the most out of post analytics in Ghost.'
                         title='Understanding analytics in Ghost'
-                        url='https://ghost.org/help/post-analytics/'>
+                        url='https://ghost.org/help/native-analytics'>
```

Comment:
Looks like this link doesn't exist yet. I wonder if it's intended.

Reply:
Thank you, yes, this is not done yet but it's coming.